AppleTVRemoteDemoApp.xcodeproj/project.pbxproj

```diff
@@ -230,20 +231,24 @@
 		4FCB9BD51BCC608E00B4F841 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				DEVELOPMENT_TEAM = 268S5GJFWZ;
 				INFOPLIST_FILE = AppleTVRemoteDemoApp/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.TechBrewers-LTD.AppleTVRemoteDemoApp";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 4.2;
```

Comment:
In Xcode 10 wouldn't this be the default?

Reply:
If you create a new project, yes it's the default.  For an existing project, however; you have a choice (the value was initially "Unspecified"). 

**Reference Screenshot**

![image](https://user-images.githubusercontent.com/2284832/46245876-e0540d80-c3b2-11e8-89ad-691546303d02.png)